scripts/build-docker.sh

```diff
@@ -15,13 +15,13 @@ source "${mydir}/utils.sh"
 
 usage() {
   msg
-  msg "Usage: $0 [-h|--help] [-p|--package hoprd|cover-traffic-daemon] [-f|--force]"
+  msg "Usage: $0 [-h|--help] [-p|--package hoprd|cover-traffic-daemon] [-f|--force] [-n|--no-tags]"
   msg
-  msg "Use -f to force a Docker build. No additional tags will be applied though."
+  msg "Use -f to force a Docker build. No additional tags will be applied though if no environment has been found."
```

Comment:
The meaning of this log message is somehow obscure to me

Reply:
done